test/lib/docker.spec.ts

```diff
@@ -93,6 +95,35 @@ describe("docker", () => {
       });
     }
 
+    test("Pass platform argument when pulling from registry", async () => {
```

Comment:
Should we move this test to its own `describe` block since the current one relates to the `docker.save(...)` command?

Reply:
great catch. thanks for the suggestion!